pseudocode

```diff
@@ -90,6 +90,49 @@ public class Project2002 {
         System.out.println();
     }
 }
+/*Newest code edited, I used the code above, divided into methods and added an if/else statment. I had another idea for a method
+But I want to get the team's approval first, this code is fully opertnational*/
```

Comment:
I also changed the file path, this program must be in the same folder as the zip with names